release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2645,3 +2645,72 @@ Issued: 2020-06-15
 An update for 	openshift-enterprise-hyperkube-container is now available for
 {product-title} 4.4. Details of the update are documented in the
 link:https://access.redhat.com/errata/RHSA-2020:2449[RHSA-2020:2449] advisory.
+
+[[ocp-4-4-9-features]]
+==== Features
+
+[id="ocp-4-4-9-ibm-power"]
+===== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power] or xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network].
+
+*Restrictions*
+
+Note the following restrictions for {product-title} on IBM Power:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
+** Container-native virtualization (CNV)
+** {product-title} Serverless
+
+* The following {product-title} features are unsupported:
+** Red Hat OpenShift Service Mesh
+** OpenShift Do (odo)
+** CodeReady Containers (CRC)
+** {product-title} Pipelines based on Tekton
+** {product-title} Metering
+** SR-IOV CNI plug-in
+
+* Worker nodes must run Red Hat Enterprise Linux CoreOS.
```

Comment:
We have an attribute for this. use `{op-system-first}` instead of "Red Hat Enterprise Linux CoreOS".

Another instance below.